lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -1034,7 +1058,7 @@ def create_servers(action_handler, specs_and_options, parallelizer, &block)
       end
 
       # Create the servers in parallel
-      parallelizer.parallelize(by_bootstrap_options) do |bootstrap_options, machine_specs|
+      parallelizer.parallel_do(by_bootstrap_options) do |bootstrap_options, machine_specs|
```

Comment:
The main difference between them: `parallelize` returns a `ParallelEnumerable` *immediately*, and stuff happens in the background (but the `to_a` should wait until everything is done). `parallel_do` waits for everything to finish before it completes, but doesn't return anything.